grpc-ballerina/Module.md

```diff
@@ -0,0 +1,328 @@
+## Overview
+
+This module provides an implementation for connecting and interacting with gRPC endpoints. 
+
+gRPC is an inter-process communication technology that allows you to connect, invoke and operate distributed heterogeneous applications as easily as making a local function call. The gRPC protocol is layered over HTTP/2 and It uses Protocol Buffers for marshaling/unmarshaling messages. This makes gRPC, highly efficient on wire and a simple service definition framework.
+
+When you develop a gRPC application the first thing you do is define a service definition using Protocol Buffers.
+
+### Protocol Buffers
+This is a mechanism to serialize the structured data introduced by Google and used by the gRPC framework. Defining the service using Protocol Buffers includes defining remote methods in the service and defining message types that are sent across the network. A sample service definition is shown below.
+
+```proto
+syntax = "proto3";
+
+service Helloworld {
+     rpc hello(HelloRequest) returns (HelloResponse);
```

Comment:
Shouldn't the standard indentation equal to 4 spaces?
Check the other instances as well.